packages/auth/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Auth",
```

Comment:
i believe this should be "@recidiviz/auth"

Reply:
Thank you so much - I was going to ask about this whole section!